### ISSUE TITLE
IE 11 compatibility

### ISSUE THREAD

**Issue by dietervu** [clarin-eric/switchboard#53]
Check why IE 11 does not render the switchboard. Probable cause as reported by Hendrik earlier for the clarin.eu website:

By chance I googled the problem and found that a missing compatibility header could cause the problem in IE-11, who defaults to compatibility-mode in intranets:

<meta http-equiv="X-UA-Compatible" content="IE=edge">

I cannot find that element in the sourcecode of the page.

This is being proposed here:

​https://stackoverflow.com/questions/22302106/pages-does-not-render-properly-in-ie-11-when-using-the-bootsrapv3-0-0-theme

**Comment by emanueldima** [comment]
Unfortunately it's a bigger problem: IE 11 does not support ECMAScript 6, which the new version of the UI is developed in. I made it work by adding polyfills, which is a small code change, but that increases the (already large) js bundle size by ~130k, from 355k to 485k. Should we go on with this, and support IE11?

**Comment by dietervu** [comment]
Yes. It is still in use at many places, like it or not. Especially since we want the switchboard to be able to function as demonstrator at any computer. When [Windows 10](https://support.microsoft.com/en-us/help/17454/lifecycle-faq-internet-explorer) is replaced we can remove the support again.
